jina/docker/checker.py

```diff
@@ -77,3 +77,13 @@ def get_exist_path(directory, s):
 
 def is_error_message(s):
     return re.search(excepts_regex, s, re.IGNORECASE | re.UNICODE) is not None
+
+
+def db_env_variables_set():
+    """ Checks if any of the db env variables are not set """
+    none_exists = None in [
+        os.environ.get('db_username', None), os.environ.get('db_password', None),
+        os.environ.get('db_hostname', None), os.environ.get('db_name', None),
+        os.environ.get('db_collection', None)
+    ]
```

Comment:
```suggestion
     keys = ['db_username', 'db_password', ...]
     for k in keys:
        if k not in os.environ:
           return False
```

also I suggest `db_username`, `db_password` to have all-caps letter and with `JINA_` prefix

Reply:
Changed this.